<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>options</class>
 <widget class="QWidget" name="options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>318</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Transfer the file rights</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="doRightTransfer">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Keep the file date</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="keepDate">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Block size</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="blockSize">
     <property name="suffix">
      <string>KB</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>64000</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Auto start the transfer</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="autoStart">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>When folder error</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="comboBoxFolderError">
     <item>
      <property name="text">
       <string>Ask</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Skip</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>When folder collision</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="comboBoxFolderColision">
     <item>
      <property name="text">
       <string>Ask</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Merge</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Skip</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rename</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Check if destination folder exists</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="checkBoxDestinationFolderExists">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Do checksum</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Quick md5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Quick sha1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Full crc32</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Full md5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Full sha1</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Enable OS buffer</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QCheckBox" name="checkBox_3">
     <property name="text">
      <string>OS buffer only if lower than</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QSpinBox" name="spinBox">
     <property name="suffix">
      <string>KB</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2048</number>
     </property>
     <property name="value">
      <number>512</number>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QPushButton" name="filters">
     <property name="text">
      <string>Filters</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="checkBox_4">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Checksulm only after error</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
